### PR TITLE
[318] Multiple excludes for binary rsync

### PR DESCRIPTION
**Pull request by adietish**
provides a fix for #318 

This is breaking the public API, thus it's suggested at 6.0.0-SNAPSHOT

### CODE REVIEW

File: src/main/java/com/openshift/restclient/capability/IBinaryCapability.java [hunk 1]
```diff
@@ -12,21 +12,24 @@
...
+		@Override
+		public void append(StringBuilder commandLine) {
+			commandLine.append(" --insecure-skip-tls-verify=true");
+		}
 	}
 	
+	abstract static class CommandLineArgument {		
+		public abstract void append(StringBuilder builder);
+	}
+
```
Comment (jeffmaury) [inline src/main/java/com/openshift/restclient/capability/IBinaryCapability.java]: Why not an interface ?

**Comment by jeffmaury** [comment]
This is a breaking change so I would favor switch to 6.x version. Also there are a couple of artifacts using this in addition to JBossTools (see https://mvnrepository.com/artifact/com.openshift/openshift-restclient-java/usages), I checked openshift-pipeline don't seem to be affected but maybe worth checking

**Comment by adietish** [comment]
@jeffmaury @jcantrill this is ready for merge, please review

File: src/main/java/com/openshift/restclient/capability/IBinaryCapability.java (hunk 1 above)
Comment (jeffmaury) [reply src/main/java/com/openshift/restclient/capability/IBinaryCapability.java]: And having it a functional interface would allow lamba to be used

**Comment by adietish** [comment]
[test]

**Comment by adietish** [comment]
once this PR is approved we should release a 5.9.6.Final which has the fix for ignoring .git folders in OpenShiftBinaryRSync since this PR is bumping openshift-restclient-java to 6.0.0-SNAPSHOT

**Comment by adietish** [comment]
@jcantrill thx!

**Comment by adietish** [comment]
[merge]

**Pull request merged**